examples/keptn/README.md

```diff
@@ -0,0 +1,120 @@
+# Evaluation & Quality Gate based promotions using Keptn
+
+## Description
+
+In this example we will show how to use Keptn to perform a promotion based on the evaluation of a Quality Gate.
+
+The Application Group deploys an application that relies on multiple supporting layers to be deployed and running before the application can be started. The *bookinfo* application relies on the following layers:
+
+- *Istio CRDs* - This layer (*istio-base*) contains the Istio CRDs that are used to configure the Istio service mesh.
+- *Prometheus* - This layer contains the Prometheus component that is used to monitor the application and leveraged by Keptn to perform the evaluation of the Quality Gate.
+- *Istio* - This layer (*istiod) )contains the Istio service mesh that is used to deploy the application.
+- *Istio Ingress Gateway* - This layer contains the Istio Ingress Gateway that is used to expose the application.
+- *Bookinfo* - This layer contains the *bookinfo* application.
+
+The scenarios that we will use in this example are:
+
+The *bookinfo* application is deployed with the Istio sidecar injection enabled. The application is configured to use the Keptn executor to perform evaluation of the Quality Gate.
+
+1. The *productpage* sidecar is configured to serve traffic without any issues.
+We expect the `Workflow` and subsequently the `ApplicationGroup` to succeed.
+
+### Keptn dashboard - Success
+
+> ⚠️ monitoring failed is a known, benign issue when submitting the `ApplicationGroup` multiple times.
+
+![Keptn Dashboard](./keptn-dashboard.png)
+
+2. The *productpage* sidecar is configured to inject faults (return status code 500, 80% of the time) using the [`VirtualService`](https://istio.io/latest/docs/tasks/traffic-management/fault-injection/). We expect the `Workflow` and subsequently the `ApplicationGroup` to fail & rollback to the previous `ApplicationGroup` spec (i.e. Scenario 1).
+
+### Keptn dashboard - Failure
+
+![Keptn Dashboard](./keptn-dashboard-failed.png)
+
+## Installation
+
+### Orkestra Helm Chart
+
+From the root directory of the repository, run:
+
+```shell
+helm upgrade --install orkestra chart/orkestra -n orkestra --create-namespace --set=keptn.enabled=true --set=keptn-addons.enabled=true  
+```
+
+### Scenario 1 : Successful Reconciliation
+
+The *bookinfo* application is deployed using the following Kubernetes manifests:
+
+```shell
+kubectl create -f bookinfo.yaml -n orkestra
+kubectl create -f bookinfo-keptn-cm.yaml -n orkestra
+```
+
+The ConfigMap is used to configure the Keptn executor and contains the following:
+
+- *keptn-config.yaml* - This file contains the Keptn configuration.
+- *sli.yaml* - This file contains the SLI configuration.
+- *slo.yaml* - This file contains the SLO configuration.
+- *config.yaml* - This file contains the configuration for the `hey` load generator.
+
+```shell
+kubectl create -f bookinfo.yaml -n orkestra
```

Comment:
Specifying this command twice is unnecessary. Also, since the commands need to be executed together, can you make this into a single command with `\`?